core/src/main/java/org/apache/iceberg/hadoop/HadoopCatalog.java

```diff
@@ -195,8 +203,108 @@ public void renameTable(TableIdentifier from, TableIdentifier to) {
     throw new UnsupportedOperationException("Cannot rename Hadoop tables");
   }
 
+  @Override
+  public boolean createNamespace(Namespace namespace) {
+    Preconditions.checkArgument(!namespace.isEmpty(), "Your Namespace must be not empty!");
```

Comment:
Error messages should contain context about what was attempted. The typical pattern used by Iceberg is "Cannot (action) (reason): (value)". For example, `"Cannot create empty namespace: %s", namespace`.

Reply:
thanks Ryan , now has fixed.